test-harness/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "test-harness"
+version = "0.1.0"
+edition = "2021"
+
+
+[[test]]
+name = "full"
```

Comment:
I'm not sure if full is the best name... But I also don't have a good idea what to use ... maybe "all"?

Reply:
Yeah, I was very happy either with `full`...
`all` is a bit better. Just got an idea: what about `toolchain` (so that we type `cargo test --test toolchain`)?